MCU_Microchip_PIC18.dcm

```diff
@@ -252,6 +252,12 @@ K Flash-Based 8-Bit Microcontroller
 F http://ww1.microchip.com/downloads/en/DeviceDoc/39564c.pdf
 $ENDCMP
 #
+$CMP PIC18F4420-xP
+D 16K Flash, 768b SRAM, 256b EEPROM, ADC, DIP-40
+K 16K Flash, 768b SRAM, 256b EEPROM, ADC
```

Comment:
The keywords field should be a little more informative than just the description again. Seems like the usual thing for this lib is "Flash-Based 8-Bit Microcontroller"